interviewing/readme.md

```diff
@@ -0,0 +1,66 @@
+
+## Step 2: The Soft Skills Interview 🥎 
+
+Soft skill interview questions are typically asked at some point during the interview process. 
+A common technique to answer these questions is to use the [STAR method](https://ca.indeed.com/career-advice/interviewing/star-interview-method) which stands for Situation, Task, Action and Result.
+
+“Can you tell me about a time when you had to work in a team and faced a challenge?”
+
+- <strong> Situation </strong>: "During my final group project, we had to develop a mobile app for managing student assignments. Midway through the project, we hit a roadblock because two teammates disagreed on which framework to use—one preferred React Native, while the other wanted to stick with Flutter. This was causing delays in our progress.”
+- <strong> Task </strong>: “As the team lead, my task was to resolve the conflict and ensure we stayed on schedule.”
+- <strong> Action </strong>: “I organized a meeting to discuss the pros and cons of both frameworks, focusing on performance and our project requirements. After evaluating, I suggested using React Native because it allowed us to quickly integrate with the backend we’d already built. The teammate supporting Flutter agreed once we reviewed the long-term benefits.”
+- <strong> Result </strong>: “We moved forward with React Native, completed the app on time, and received positive feedback from our professor for both functionality and performance. It also helped the team work more smoothly on future features.”
+
+Discuss how you might answer the following questions in a real interview.
+
+- Tell us about a time you missed a deadline.
+- What was your favorite project you have worked on so far.
+- When was a time you failed.
+- Tell us about a time you had to help your team members through a tough time. 
+- Tell me about a time you prioritised your time well, despite having multiple deadlines.
+- Tell me about a time you solved a problem at work.
+- How do you teach your colleagues about new practices within the company?
+- Tell me about a time when things didn't go according to plan. How did you deal with it?
+- What is your response when a colleague disagrees with you?
+- Share a time you set a goal and achieved it.
+
+## Step 3: The Technical Interview 📞
+
+Pick any technical topic such as Javascript, Python, SQL, databases, etc. and search for the Top 50 beginner interview questions for that topic. Read through the list together. Discuss if there are any surprises in there. Which questions did you find easy and which ones did you find difficult? Try asking each other a couple questions and see if you can answer them.
+
+Some tips for technical interviews:
+- Think out loud, walk the interviewer through your thought process.
+- Interviewers do not expect you to know everything. Be honest when you do not know something.
+- If you do not understand something, ask questions!
+- Practice, practice, practice
```

Comment:
Let's also get a summary at the end. How will we complete this workshop? What are the takeaways? Where should we go to learn more and practice what we have learned?

Add in 2-3 summary sentences and then a takehome task like "book a call to practice an interview by the end of this week" "volunteer to act as interviewer by the end of this sprint" etc

Reply:
Added 🚀